### PR TITLE
My Portfolio

### PR DESCRIPTION
**Pull request by ridaarif98**
In Porfoli,
I create a markup page
Linke markup page with style sheet
Add header and section in the markup page
Use flex for header and section
Integrate linters
Correct linter error and update

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    <h1 class="myLogo">MY LOGO</h1>
+    <i class="fa fa-bars" ></i>
+  </header>
+
+  <section class="headlineSection">
+      <div class="imageDiv">
+        <img  class="firstImg" src="images/Ellipse11.png" alt="Ellipse" >
+        <img class="secondImg" src="images/Rectangle7.png" alt="Rectangle">
+        <img class="thirdImg" src="images/Rectangle4.png" alt="Rectangle"> 
+        <img class="fourthImg" src="images/Ellipse3.png" alt="Elipse">
```
Comment (DeliceLydia) [inline Index.html:22]: - [ ] Decorations are supposed to be background image not img tag as instructions say please fix that

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    </div>
+    <div class="detailSection">
+            <h3>I'm Rida Glad to see you!</h3>
```
Comment (DeliceLydia) [inline Index.html:25]: - [ ] Main title is a h1 tag not h3 please fix that

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        <a href="#">LET'S CONNECT</a>
+        <br>        
+
+    </div>
+    <div class="links">
+        <a><span class="fa fa-twitter" ></span></a>
+      <a><span class="fa fa-angellist"></span></a>
+        <a><span class="fa fa-github"></span></a>  
+     <a><span class="fa fa-linkedin"></span></a>
+    </div>
```
Comment (DeliceLydia) [inline Index.html:43]: - [ ] Please use ul tag to group icons and li tag per icon and each icon has a link

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            <h3>I'm Rida Glad to see you!</h3>
+    </div>
+    <div class="discription">
+        <p>
+            I'm a software developer! I can help you build a product,
+             feature, website Look through some of my Work and experience! If you
+             like what you see and have a project you need coded, don't hesitate 
+             to contact me.
+        </p>
+        <a href="#">LET'S CONNECT</a>
```
Comment (DeliceLydia) [inline Index.html:34]: - [ ] on the button you need to follow the given design on button styling ( active, hover, pressed)

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hello @ridaarif98 ✋, 

Wow good job @ridaarif98, I like your design so far your project looks good already. However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Practice the DRY rule in your CSS you can do it by grouping items that have the same properties and give them the same class 

## Cheers and Keep Rocking! 💪

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+   <title>My Portfolio</title>
+</head>
+<body>
+  <header class="header">
+    <h1 class="myLogo">MY LOGO</h1>
+    <i class="fa fa-bars" ></i>
+  </header>
```
[the remark this answers is not in the document]
Comment (descholar-ceo) [reply Index.html]: :+1: These two issues have been addressed correctly :heavy_check_mark:

File: Index.html (hunk 1 above)
Comment (descholar-ceo) [reply Index.html]: :+1: This issue has been addressed correctly :heavy_check_mark:

File: Index.html (hunk 2 above)
Comment (descholar-ceo) [reply Index.html]: :+1: This issue has been addressed correctly :heavy_check_mark:

File: Index.html (hunk 3 above)
Comment (descholar-ceo) [reply Index.html]: :+1: This issue has been addressed correctly :heavy_check_mark:

File: Index.html (hunk 4 above)
Comment (descholar-ceo) [reply Index.html]: :+1: This issue has been addressed correctly :heavy_check_mark:

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @ridaarif98 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**